library/std/src/diagnostics.qs

```diff
@@ -69,11 +69,37 @@ namespace Microsoft.Quantum.Diagnostics {
         body intrinsic;
     }
 
+    /// # Summary
+    /// Checks whether a qubit is in the |0⟩ or ground state, returning true if it is.
+    ///
+    /// # Description
+    /// This operation checks whether a qubit is in the |0⟩ or ground state. It will return true only
+    /// if the qubit is deterministically in the |0⟩ state, and will return false otherwise. This operation
+    /// does not change the state of the qubit.
```

Comment:
Maybe have a remarks section that such check wouldn't be possible on an actual hardware?

Reply:
Added